update.go

```diff
@@ -0,0 +1,191 @@
+// +build linux
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/codegangsta/cli"
+)
+
+type updateRessources struct {
```

Comment:
I think it's a better idea to use the OCI spec fields, since runC is meant to be the example OCI runtime implementation. We could check that the user hasn't set any other fields in the config (and error out in that case), but then we wouldn't add a slightly incompatible (and possibly out of date in the future) JSON payload for updating resources.

Also, you typo'd the word `Resources` in `updateRessources`.